elements/includes/creative_commons.inc

```diff
@@ -558,6 +558,11 @@ function xml_form_elements_creative_commons_get_values_from_uri($license_uri) {
     xml_form_elements_creative_commons_make_value_array('manual', '', '', '', '', $license_uri)
   );
 
+  // If no license URI is set, default to no output.
+  if (empty($license_uri)) {
+    $values['license_fieldset']['license_type'] = 'none';
```

Comment:
nice

Reply:
@d-r-p 
> When users choose to clean up their MODS from empty CC elements - as we suggest, actually - editing such objects will result in the '#default_value' being loaded rather than "no license". Should we fix/document this?

Not sure if that is an issue or how can that be fixed. Assuming the Administrator builds an XML form with a given default for CC, the default being loaded everytime the form is invoked without a given XML element seems expected. I mean the 'none' means NO license or means no XML element? If we define that well in docs and we don't break existing/expected functionality it should be good. I feel this could end being over-engineered and will eventually break anyway in the presence of a wrongly set :read :update xpath 😄

Reply:
@DiegoPino I have not tested this, but I have a feeling this is a general "issue" (i.e. pertaining to all form elements), anyway. Your comment seems to confirm this.

Regarding the _expected_ behaviour: I am not sure what to think. If the element remains empty, the `'#default_value'` will not be introduced. If the element is nuked, the element will populate with the `'#default_value'` upon edit. Since both cases stem from submitting the element with no value, I see an inconsistency for users. But, perhaps, I am overthinking this... In any case, this seems like a separate issue that can be discussed further elsewhere, right?

Reply:
> Since both cases stem from submitting the element with no value, I see an inconsistency for users.

I agree

>  In any case, this seems like a separate issue that can be discussed further elsewhere, right?

Also true. We can only save one tree at a time and lately pulls are getting too big all over the world. Single pulls are easier to address and to review. So ++ on moving forward. Will do a review later today. Thanks again